{{cookiecutter.buildfolder}}/dump_datamodel.py

```diff
@@ -23,10 +23,28 @@
 
 if __name__ == "__main__":
     if len(sys.argv) != 4:
-        print("Usage: ./dump_datamodel.py <space> <data_model> <directory>")
+        print(
+            "Usage: ./dump_datamodel.py <space|all|global> <data_model|all> <directory>"
+        )
         exit(1)
     if not os.path.exists(sys.argv[3]):
         os.makedirs(sys.argv[3])
+    if sys.argv[1] in ["all", "global"]:
+        if sys.argv[1] == "global":
+            include_global = True
+        else:
+            include_global = False
+        if sys.argv[2] == "all":
+            model_name = None
+        else:
+            model_name = sys.argv[2]
+        dump_datamodels_all(
+            ToolGlobals,
+            target_dir=sys.argv[3],
+            model_name=model_name,
+            include_global=include_global,
+        )
+        exit(0)
```

Comment:
No big deal, but there are some quite nice libraries for this

Reply:
Yeah, I know. I just hate adding dependenices for super-simple things. Could have used click, but I think we need to iterate on this and as we discussed, push this into the config/dump service.